desktop-src/shell/executing-control-panel-items.md

```diff
@@ -52,6 +52,67 @@ This topic discusses the following:
+
+
+```
+WinExec("%systemroot%\system32\control.exe /name Microsoft.ProgramsAndFeatures", SW_NORMAL);
+```
+
+
+
+To start a Control Panel item with its canonical name, use: "%systemroot%\\system32\\control.exe /name *canonicalName*"
+
+To open a specific sub-page in an item, or to open it with additional parameters, use: "%systemroot%\\system32\\control.exe /name **canonicalName** /page **pageName**"
+
+An application can also implement the [**IOpenControlPanel::Open**](/windows/desktop/api/shobjidl_core/nf-shobjidl_core-iopencontrolpanel-open) method to launch Control Panel items, including the ability to open a specific sub-page.
+
+For a complete list of Control Panel item canonical names, see [Canonical Names of Control Panel Items](controlpanel-canonical-names.md).
+
+## Command overview
+
+Windows 11 contains the folowwing control panel items:
+
+### Personalization
+
+-   Screensaver: %windir%\\system32\\control.exe desk.cpl,screensaver,@screensaver
+
+### System
+
+-   Performance: `%windir%\\system32\\SystemPropertiesPerformance.exe`
+-   Remote access: `%windir%\\system32\\SystemPropertiesRemote.exe`
+-   Computer name: `%windir%\\system32\\SystemPropertiesComputerName.exe`
+-   System protection: `%windir%\\system32\\SystemPropertiesProtection.exe`
+-   Advanced system properties: `%windir%\\system32\\SystemPropertiesAdvanced.exe`
+
+### Programs and Features
+
+-   Add or remove programs: `%windir%\\system32\\control.exe /name Microsoft.ProgramsAndFeatures`
+-   Windows features: `%windir%\\system32\\OptionalFeatures.exe`
+
+### Folder Options
+
+-   View: `%windir%\\system32\\rundll32.exe shell32.dll,Options_RunDLL 7`
+-   General: `%windir%\\system32\\rundll32.exe shell32.dll,Options_RunDLL 0`
+
+### Power Options
+
+-   Edit current plan settings: `%windir%\\system32\\control.exe /name Microsoft.PowerOptions /page pagePlanSettings`
+-   System settings: `%windir%\\system32\\control.exe /name Microsoft.PowerOptions /page pageGlobalSettings`
+-   Create a power plan: `%windir%\\system32\\control.exe /name Microsoft.PowerOptions /page pageCreateNewPlan`
+-   Power Options: `%windir%\\system32\\control.exe powercfg.cpl,,3`
+
+## Legacy Control Panel Commands
+
 ## Windows Vista Canonical Names
 
 In Windows Vista and later, the preferred method of launching a Control Panel item from a command line is to use the Control Panel item's canonical name. A canonical name is a non-localized string that the Control Panel item declares in the registry. The value of using a canonical name is that it abstracts the module name of the Control Panel item. An item can be implemented in a .dll and later be reimplemented as a .exe or change its module name. As long as the canonical name remains the same, then any program that opens it by using that canonical name does not need to be updated.
```

Comment:
This paragraph is now repeated from line 60. It could make sense to update this paragraph to instead mention that the method for launching Control Panel items with its canonical name in Windows Vista and later is the same as it is for Windows 11.